packages/python/javdb_spider/fetch/session.py

```
@@ -198,6 +202,95 @@ def attempt_login_refresh(explicit_proxies=None, explicit_proxy_name=None,
+            "[%s] Verifying login via fixed page: %s", label, url,
+        )
+        try:
+            html = handler.get_page(
+                url,
+                use_cookie=True,
+                use_proxy=True,
+                module_name='spider',
+                max_retries=2,
+                use_cf_bypass=use_cf_bypass,
+            )
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)